nerdlandbot/commands/pets.py

```diff
@@ -0,0 +1,309 @@
+import typing
+import discord
+from discord.ext import commands
+import asyncio
+import random
+from PIL import Image
+import io
+from pathlib import Path
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.commands.GuildData import get_guild_data, GuildData
+from nerdlandbot.helpers.constants import PETS_DIR_NAME, IMAGE_SIZE, INTERACT_TIMEOUT
+
+
+class Pets(commands.Cog, name="Pets"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        name="add_pet",
+        brief="add_pet_brief",
+        help="add_pet_help",
+        usage="add_pet_usage",
+    )
+    @commands.guild_only()
+    async def add_pet(
+        self,
+        ctx: commands.Context,
+        name: typing.Optional[str] = None,
+        category: typing.Optional[str] = None,
+    ):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        if name is None:
+            message = translate("pet_need_name", lang)
+            return await ctx.send(message)
+        if category is None:
+            message = translate("pet_need_category", lang)
+            return await ctx.send(message)
+
+        if category.lower() not in guild_data.pets_categories:
+            message = translate("pet_category_nonexistant", lang)
+            return await ctx.send(message)
+
+        
+        await guild_data.add_pet(name, ctx.author.id, category.lower())
+        pet_id = guild_data.pets_last_id
+        if ctx.message.attachments:
+            await resize_and_save(ctx, ctx.message.attachments[0], pet_id)
+            msg = translate("pet_added_succes", lang).format(
+                name.capitalize(), pet_id)
+            await ctx.send(msg)
+        else:
```

Comment:
Remove the else statement here and lose the indentation. Return inside the previous if.